src/scripts/fetch-routes.ts

```diff
@@ -60,6 +60,10 @@ const dynamicRoutesConfig: RouteConfig[] = [
     queryOperation: "allJobs",
     path: "/jobs/",
   },
+  {
+    queryOperation: "allLandingPages",
+    path: "",
```

Comment:
I could also make the `path` optional on the type, would you prefer that?

Reply:
Nah I prefer it to be explicitly empty